packages/sit-onyx/src/components/OnyxDataGrid/features/index.ts

```diff
@@ -219,15 +219,41 @@ export const useDataGridFeatures = <
     header: { component: HeaderCell },
     cell: { component: (props) => String(props.modelValue) },
   };
+
+  const numberFormatter = (value: TEntry[keyof TEntry] | undefined) => {
+    if (typeof value !== "number") {
+      return;
```

Comment:
Should we really show nothing if its no number? Maybe we could use the fallbackRenderer instead